src/cdk-experimental/popover-edit/edit-event-dispatcher.ts

```diff
@@ -88,20 +182,120 @@ export class EditEventDispatcher {
     this._editRef = null;
   }
 
+  /** Adds the specified table row to be tracked for first/last row comparisons. */
+  registerRowWithHoverContent(row: Element): void {
+    this._rowsWithHoverContent.set(row, (this._rowsWithHoverContent.get(row) || 0) + 1);
+  }
+
+  /**
+   * Reference decrements and ultimately removes the specified table row from first/last row
+   * comparisons.
+   */
+  unregisterRowWithHoverContent(row: Element): void {
```

Comment:
`unregister` -> `deregister` (I just realized we use the latter elsewhere and want to keep it consistent)